modules/UpgradeWizard/language/en_us.lang.php

```diff
@@ -85,8 +85,8 @@
     'LBL_UW_CANCEL_DESC' => 'The upgrade has been cancelled. Any temporary files that were copied and any upgrade files that were uploaded have been deleted.',
     'LBL_UW_CHECK_ALL' => 'Check All',
     'LBL_UW_CHECKLIST' => 'Upgrade Steps',
-    'LBL_UW_COMMIT_ADD_TASK_DESC_1' => "Backups of Overwritten Files are in the following directory: \n",
-    'LBL_UW_COMMIT_ADD_TASK_DESC_2' => "Manually merge the following files:\n",
+    'LBL_UW_COMMIT_ADD_TASK_DESC_1' => 'Backups of Overwritten Files are in the following directory: \n',
+    'LBL_UW_COMMIT_ADD_TASK_DESC_2' => 'Manually merge the following files:\n',
```

Comment:
In some cases the newline would not display correctly when in a single quote versus a double quote. Thoughts about closing the quote and using `PHP_EOL`

Reply:
@isleshocky77 Looks like this was missed. Thanks! I've made a new PR here: #4861